.github/workflows/e2e-test.yml

```diff
@@ -36,24 +36,19 @@ jobs:
       fail-fast:     false
       matrix:
         # Min WooCommerce | L-2 | L(atest) | beta
-        woocommerce:   [ '6.0.1', '6.5.1', 'latest', 'beta' ]
+        woocommerce:   [ '6.2.2', '6.6.0', 'latest', 'beta' ]
```

Comment:
If we change `WC 6.6.0` to `WC 6.6.1` in [this comment](https://github.com/Automattic/woocommerce-payments/pull/4524/files#r942407608), should we also update it here?

Reply:
Fixed in [094e3f3](https://github.com/Automattic/woocommerce-payments/pull/4524/commits/094e3f3b9008161e6342fc1153ad975e9fee2125)